Source/dom/layers/Text.js

```diff
@@ -36,6 +36,19 @@ export const TextAlignmentMap = {
   natural: 4, // Indicates the default alignment for script
 }
 
+// Mapping between vertical text alignment names and values.
+const VerticalTextAlignment = {
+  top: 'top', // Visually top aligned
+  center: 'center', // Visually center aligned
+  bottom: 'bottom', // Visually bottom aligned
+}
+
+export const VerticalTextAlignmentMap = {
+  left: 0, // Visually left aligned
```

Comment:
Is this `left` correct? Shouldn't this be `top`?

Reply:
Whoops, missed that when adjusting after copying and pasting.